core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/rabbitmq.yml

```diff
@@ -1,30 +1,38 @@
 ---
-- name: RabbitMQ | Check version
-  command: rabbitmqctl version
-  register: rabbitmq_version_current
+- include_tasks: "rabbitmq/check-version-{{ ansible_os_family | lower }}.yml"
+
+# There is no simple way to check if nodes are clustered
+# so the number of running nodes is checked
+- name: RabbitMQ | Check cluster running nodes
+  shell: rabbitmqctl cluster_status | sed -n '/Running\ Nodes/,/Versions/{//!p;}'
```

Comment:
How about parsing json payload? :thinking: 
```shell
root@u1a2:~# rabbitmqctl cluster_status --formatter json
{"alarms":[],"cluster_name":"rabbit@u1a2","disk_nodes":["rabbit@u1a2","rabbit@u1a3","rabbit@u1b2"],"feature_flags":[{"desc":"Default bindings are now implicit, instead of being stored in the database","doc_url":"","name":"implicit_default_bindings","provided_by":"rabbit","stability":"stable","state":"enabled"},{"desc":"Maintenance mode status","doc_url":"","name":"maintenance_mode_status","provided_by":"rabbit","stability":"stable","state":"enabled"},{"desc":"Support queues of type `quorum`","doc_url":"https://www.rabbitmq.com/quorum-queues.html","name":"quorum_queue","provided_by":"rabbit","stability":"stable","state":"enabled"},{"desc":"Virtual host metadata (description, tags, etc)","doc_url":"","name":"virtual_host_metadata","provided_by":"rabbit","stability":"stable","state":"enabled"}],"listeners":{"rabbit@u1a2":[{"interface":"[::]","node":"rabbit@u1a2","port":25672,"protocol":"clustering","purpose":"inter-node and CLI tool communication"},{"interface":"[::]","node":"rabbit@u1a2","port":5672,"protocol":"amqp","purpose":"AMQP 0-9-1 and AMQP 1.0"}],"rabbit@u1a3":[{"interface":"[::]","node":"rabbit@u1a3","port":25672,"protocol":"clustering","purpose":"inter-node and CLI tool communication"},{"interface":"[::]","node":"rabbit@u1a3","port":5672,"protocol":"amqp","purpose":"AMQP 0-9-1 and AMQP 1.0"}],"rabbit@u1b2":[{"interface":"[::]","node":"rabbit@u1b2","port":25672,"protocol":"clustering","purpose":"inter-node and CLI tool communication"},{"interface":"[::]","node":"rabbit@u1b2","port":5672,"protocol":"amqp","purpose":"AMQP 0-9-1 and AMQP 1.0"}]},"maintenance_status":{"rabbit@u1a2":"not under maintenance","rabbit@u1a3":"not under maintenance","rabbit@u1b2":"not under maintenance"},"partitions":{},"ram_nodes":[],"running_nodes":["rabbit@u1b2","rabbit@u1a3","rabbit@u1a2"],"versions":{"rabbit@u1a2":{"erlang_version":"23.1.4","rabbitmq_name":"RabbitMQ","rabbitmq_version":"3.8.9"},"rabbit@u1a3":{"erlang_version":"23.1.4","rabbitmq_name":"RabbitMQ","rabbitmq_version":"3.8.9"},"rabbit@u1b2":{"erlang_version":"23.1.4","rabbitmq_name":"RabbitMQ","rabbitmq_version":"3.8.9"}}}
```